src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx

```diff
@@ -42,23 +40,11 @@ const CVEFormContent: React.FC<CVEFormContentProps> = ({ modalToggle }) => {
           </TextContent>
         </StackItem>
         <StackItem>
-          <InputField data-test="cve-issue-key" label="CVE key" name="key" required />
+          <InputField data-test="cve-issue-key" label="CVE ID" name="key" required />
```

Comment:
Shouldn't you also change the name and test id?

Reply:
Updated